CHANGELOG.md

```diff
@@ -3,16 +3,86 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+[#446](https://github.com/plotly/dash-table/pull/446)
+- Table API rework
+#### NEW
+    - `column.sort_as_null`: Allows sorting behavior customization.
+        Accepts an array of string, number or booleans.
+
+#### REMOVED
+    - `column.clearable`: Allows clearing the value of a dropdown cell.
+        Removed in favor of `dropdown_**` `clearable` nested property
+    - `column.options`
+        Removed. Redundant with `dropdown`.
+    - `pagination_settings`
+        Replaced by two props `page_current` and `page_size`.
+
+#### RENAMED
+    - `column_static_tooltip`
+        Renamed `tooltip`.
+    - `column_conditional_tooltips`
+        Renamed `tooltip_conditional`.
+    - `filter`
+        Renamed `filter_query`.
+    - `sort_type`
+        Renamed `sort_mode`.
+    - `derived_filter_structure`
+        Renamed to `derived_filter_query_structure`.
+
+#### MODIFIED
+    - `column.deletable`: Allows column deletion.
+        Now accepts a boolean or an array of booleans (for multi-line headers)
+    - `column.editable_name`: Allows column renaming.
+        Renamed to `column.renamable`
+        Now accepts a boolean or an array of booleans (for multi-line headers)
```

Comment:
```suggestion
        Now accepts a boolean or an array of booleans (for multi-line headers)
        For example, if there are multiple headers and you want the second row's header's name to be editable, this would be `[False, True]`
```